### ISSUE TITLE
Permanent error message - DB is not “available”

### ISSUE THREAD

**Issue by O35dE** [keepassium/KeePassium#357]
For databases that are not stored on the iphone, such as users who prefer to store them in an external encrypted folder, as an additional measure of privacy and security, a permanent and unnecessary error message is presented in red under their name saying that the file is not “available”, even if they are at rest. This message, in addition to the inconvenience, presents some information about the place of origin from which it was last accessed. 
Would it be possible to delete this message, not least because this information is naturally known to the user who chose to save the database in a place apart from the iphone, leaving it to present it, if the case, when the file is actually triggered, that is, the user tries to access but for any reason it cannot be found, in other words, just when the file is not actually available?

**Comment by keepassium** [comment]
I'm afraid I would need some specifics to understand the issue. What _exactly_ does the message say, verbatim? What is the "external encrypted folder", how can we test it?

> Would it be possible to delete this message, not least because this information is naturally known to the user who chose to save the database in a place apart from the iphone, leaving it to present it, if the case, when the file is actually triggered, that is, the user tries to access but for any reason it cannot be found, in other words, just when the file is not actually available?

This is a 72-word sentence. I cannot parse that, please rephrase.

**Comment by O35dE** [comment]
The message is displayed in red under the name of the database and in relation to one of my DBs it specifically says that "Cyptomator is not available. Please check if ...".

Ok sry, I will rephrase.

Would it be possible to eliminate this permanent “error” message in red, in relation to the databases that are stored in a external place i.e. a cryptomator folder, particullarly when they are on stand by ? 

The reason for this request is that this information is naturally known to the user who chose for some reason to save the database in a place apart from the Files app in the iPhone.

When this same user intends to open this database, he will naturally make the connections that allow the application access to the DB, so a permanent message in this regard is unnecessary while the database is on stand by.

**Comment by keepassium** [comment]
Thank you for the clarification.

> In addition, this error message contains information, even if partial, about the location of the DB (that the user chose to hide), which would not otherwise be easily identifiable.

Exactly for this reason KeePassium has app protection (app settings → App Protection → Enable AppLock). It ensures that only you can see the list of files, app settings, etc. 

> Would it be possible to eliminate this permanent “error” message in red, in relation to the databases that are stored in a external place i.e. a cryptomator folder, particullarly when they are on stand by ?

Well, KeePassium has to report that there is a problem with the database, and it should provide enough info for the user to identify and fix the problem. So no message would be bad (hides the problem) and generic message would be bad, too (no info about the cause).

The reason it is permanent is because of how Cryptomator integrates with the system. Whenever you mount the vault, it becomes a new storage location (even though the name is still "Cryptomator"). The dismounted one is gone forever.

> a simple disconnection symbol or even by some sentence but that does not mention the place or application in which the DB is stored?

File location would still remain visible via Databases → long-press database → File Info → File Location.

In summary, if you want to protect info about file location, activate KeePassium app protection.

**Comment by O35dE** [comment]
Thanks for the reply, well I am certainly aware of your app protection, but KeePassium’s solution do not delete local db files and all references to remote files, like I suggest here - https://github.com/keepassium/KeePassium/issues/356 , it will simply **close** the DBs and delete all keys from the keychain .
Thanks anyway.

**Comment by keepassium** [comment]
> well I am certainly aware of your app protection, but KeePassium’s solution do not delete local db files and all references to remote files

Just to be clear, nuking the data is a separate topic (which you filed as #356). 
#356 is unrelated to #357 (current issue, hiding file location)
App protection does cover #357, so let's consider this one solved.
